CameraOverlay/research_and_development.py

```diff
@@ -0,0 +1,51 @@
+from PIL import Image, ImageDraw, ImageFont
+
+WIDTH = 1280
+HEIGHT = 740
+FONT = '/Users/harsilpatel/Coding/MHP_Raspicam/CameraOverlay/LibreCaslonText-Regular.ttf'
```

Comment:
Think we should get rid of hard coded paths like these

Reply:
yeah sorry, it was more of a quick-n-dirty script initially, but fixed it ;)